Lib/abc.py

```diff
@@ -3,7 +3,28 @@
 
 """Abstract Base Classes (ABCs) according to PEP 3119."""
 
-from _weakrefset import WeakSet
+
+def get_cache_token():
+    """Returns the current ABC cache token.
+
+    The token is an opaque object (supporting equality testing) identifying the
+    current version of the ABC cache for virtual subclasses. The token changes
+    with every call to ``register()`` on any ABC.
+    """
+    return ABCMeta._abc_invalidation_counter
+
+
+_C_speedup = False
+
+try:
+    from _abc import (get_cache_token, _abc_init, _abc_register,
+                      _abc_instancecheck, _abc_subclasscheck, _get_dump,
+                      _reset_registry, _reset_caches)
+except ImportError:
+    # We postpone this import to speed-up Python start-up time
+    from _weakrefset import WeakSet
```

Comment:
This import needs to be done even if the C accelerator is available (or be done lazily), because otherwise the `_Py_AbcMeta` class exists but is broken.